docs/pages/components/dialog.md

```diff
@@ -0,0 +1,244 @@
+</div>
+{% endcapture %}
+
+{% include display-component.html component=success-dialog %}
+
+<br/>
+
+<br/>
+
+## Medium Error Dialog
+This is used to present error message to the user 
+
+{% capture error-dialog %}
+<div class="modal-demo-bg">
+<div class="fd-dialog--medium" role="dialog">
+  <div class="fd-dialog__content" role="document">
+      <header class="fd-dialog__header-error">
+          <h1 class="fd-dialog__title sap-icon--message-error">Error Message</h1>
+      </header>
+    <div class="fd-dialog__body">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.</p>
+    </div>
+    <footer class="fd-dialog__footer">
+        <div class="fd-dialog__actions">
+         <button class=" fd-button--emphasized">Confirm</button>
+          <button class=" fd-button--secondary">Cancel</button>
+        </div>
+    </footer>
+  </div>
+
+</div>
+</div>
+{% endcapture %}
+
+{% include display-component.html component=error-dialog %}
+
+<br/>
+## Medium Warning Dialog
+This is used to present warning message to the user 
+
+{% capture warning-dialog %}
+<div class="modal-demo-bg">
+<div class="fd-dialog--medium" role="dialog">
+  <div class="fd-dialog__content" role="document">
+      <header class="fd-dialog__header-warning">
+          <h1 class="fd-dialog__title sap-icon--message-warning">Warning Message</h1>
+      </header>
+    <div class="fd-dialog__body">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.</p>
+    </div>
+    <footer class="fd-dialog__footer">
+        <div class="fd-dialog__actions">
+         <button class=" fd-button--emphasized">Confirm</button>
+          <button class=" fd-button--secondary">Cancel</button>
+        </div>
+    </footer>
+  </div>
+
+</div>
+</div>
+{% endcapture %}
+
+{% include display-component.html component=warning-dialog %}
+
+<br/>
+
+## Medium Success Dialog
+This is used to present success message to the user 
+
+{% capture success-dialog %}
+<div class="modal-demo-bg">
+<div class="fd-dialog--medium" role="dialog">
+  <div class="fd-dialog__content" role="document">
+      <header class="fd-dialog__header-success">
+          <h1 class="fd-dialog__title sap-icon--message-success">Success Message</h1>
+      </header>
+    <div class="fd-dialog__body">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.</p>
+    </div>
+    <footer class="fd-dialog__footer">
+        <div class="fd-dialog__actions">
+         <button class=" fd-button--emphasized">Confirm</button>
+          <button class=" fd-button--secondary">Cancel</button>
+        </div>
+    </footer>
+  </div>
+
+</div>
+</div>
+{% endcapture %}
+
+{% include display-component.html component=success-dialog %}
+
+<br/>
+
+## Phone full screen
+This is used to present message on phone
```

Comment:
There shouldn't be a specific type of dialog for phones. This can be done responsively. This approach would require scripting to swap the class name based on the device which is not necessary.